src/App.css

```diff
@@ -1,38 +1,197 @@
+
+.clearfix:after {visibility: hidden; display: block; font-size: 0; content: ''; clear: both; height: 0; }
+
+html { -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing: border-box;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+body {
   text-align: center;
+  background-color: #fffff5;
+  font-family: 'Puritan', sans-serif;
+  font-weight: 300;
+  margin: 0 auto;
+  font-size: 125%;
+  line-height: 1.5rem;
+  accent-color: #5db369;    
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+h1, h2 {
+  font-family: 'Paytone One', sans-serif;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+
+.wrapper {
+  width: 80%;
+  margin: 0 auto;
+  max-width: 1024px;
 }
 
-.App-header {
-  background-color: #282c34;
+
+header {
+  padding-top: 10px;
+}
+
+.all {
   min-height: 100vh;
   display: flex;
   flex-direction: column;
-  align-items: center;
+}
+
+main {
+  flex-grow: 1;
+}
+
+
+h1 {
+  font-size: 2.5rem;
+  font-weight: 500;
+  text-transform: uppercase;
+  margin-top: 10px;
+  margin-bottom: 30px;
+  line-height: 2.5rem;
+}
+
+
+form {
+  font-family: 'Hind Siliguri', sans-serif;
+  font-size: 0.9rem;
+  padding-bottom: 30px;
+}
+
+fieldset {
+  border: solid #dd7675 4px;
+  border-radius: 5px;
+  display: flex;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  flex-wrap: wrap;
+  margin: 20px 0;
+}
+
+legend {
```

Comment:
I love the positioning of the <legend> elements! I never would have thought of having them straddle the border top like this. I actually... can't figure it out even look in dev tools... Help?!